tests/phpunit/tests/functions/_wpCheckExistingFileNames.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @group functions.php
+ *
+ * @covers ::_wp_check_existing_file_names
+ */
+class Tests_Functions__wp_check_existing_file_names extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 55192
+	 *
+	 * @dataProvider data__wp_check_existing_file_names
```

Comment:
Fix the Unit tests error.

```suggestion
	 * @dataProvider data_wp_check_existing_file_names
```